gnomad/resources/grch38/gnomad.py

```diff
@@ -549,16 +558,9 @@ def gnomad_gks(
         if custom_coverage_ht:
             coverage_ht = custom_coverage_ht
         else:
-            # in v4, only exomes have coverage
-            # in v3, only genomes have coverage (only genomes exist)
-            if high_level_version == "v3":
-                coverage_ht = hl.read_table(
-                    coverage("genomes").versions[coverage_version].path
-                )
-            else:
-                coverage_ht = hl.read_table(
-                    coverage("exomes").versions[coverage_version].path
-                )
+            coverage_ht = hl.read_table(
+                coverage(data_type).versions[coverage_version].path
```

Comment:
@klaricch this will use `genomes` version `3.0.1` coverage table for v3 and v4 genomes tables, and `exomes` version `4.0` coverage table for v4 exomes tables. If the input is a v3 table with `data_type=exomes`, line 562 will throw an exception `KeyError: '3.0.1'`, which is expected.